### PR TITLE
Refactory cryptography util - closes #516

### LINKED ISSUE #516: Refactor cryptography util
**Expected behavior:** We should only use classes where it's necessary to track state. Also we should use consistent naming.

**Actual behavior:** We use a class for our cryptography util, but there's no state to track. This could be refactored to be a set of pure functions. We use `cryptography`, `Crypto` and `liskCrypto`.

### Versions

1.0.0

### PR DESCRIPTION
**Pull request by yatki**
Continuation of this PR: https://github.com/LiskHQ/lisk-commander/pull/587

### Review checklist

* The PR resolves #516 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -76,7 +78,7 @@
 		"chalk": "2.3.0",
 		"cli-table2": "LiskHQ/cli-table2#187c6ae80b9d963c598d0cb3379153387bfb15c4",
 		"inquirer": "6.0.0",
-		"lisk-elements": "1.0.0-beta.3",
+		"lisk-elements": "1.0.0-rc.0",
```
Comment (shuse2) [inline package.json]: I think `rc.1` is already out, so i think we can use the latest one

File: package.json [hunk 2]
```diff
@@ -57,7 +57,9 @@
 			"@oclif/plugin-help"
 		],
 		"topics": {
-			"config": { "description": "Manages Lisk Commander configuration." }
+			"config": {
+				"description": "Manages Lisk Commander configuration."
+			}
```
Comment (willclarktech) [inline package.json]: This change seems unrelated.

Comment (shuse2) [reply package.json]: prettify seems to have some random behavior.
anyone know why?

Comment (yatki) [reply package.json]: @willclarktech @shuse2 I realized when you run npm install it formats package.json. I upgraded lisk-elements with npm command thus it introduced this change. I say we should keep this.

Comment (willclarktech) [reply package.json]: Ah, with npm v5 it preserves indentation as well I think. OK!

**Comment by willclarktech** [comment]
> it's still external library, so it might be good to have wrapper 

A good point in general, but I don't think we use many of these functions more than once.
